src/file_extraction.h

```diff
@@ -37,6 +38,23 @@ void ExtractFile(const char* fileName)
     {
         code.back().push_back(tokens[a]);
 
+        bool flip;
+        if (tokens[a].type == TokenType::IMPORT || flip)
+        {
+        if (!flip)
+            continue;
+        flip = !flip;
+        std::stringstream tik;
+        tik << "./dusk_pkgs/" << tokens[a].value.cstr();
+        codeFile = Read(tik.str().c_str());
+        while (getline(codeFile, fileLine))
+        {
+            night::array<Token> temp = Lexer(fileLine.c_str());
+            for (int a = 0; a < temp.length(); ++a)
+                tokens.push_back(temp[a]);
+        }
+        }
```

Comment:
what is this indentation

Reply:
@DynamicSquid welp, looks like vs code is absolute s*** then

Reply:
wdym its just copy and paste?

Reply:
No, I have VIM keymaps. `<<` and `>>` are not working as expected on VSCode

Reply:
oh okay. could you fix it though?